Lib/test/test_exceptions.py

```diff
@@ -2011,16 +2012,20 @@ def test_copy_pickle(self):
                 self.assertEqual(exc.path, orig.path)
 
 
+def run_script(source):
+    if isinstance(source, str):
+        with open(TESTFN, 'w', encoding='utf-8') as testfile:
+            testfile.write(dedent(source))
+    else:
+        with open(TESTFN, 'wb') as testfile:
+            testfile.write(source)
```

Comment:
Should this be replaced by a call to script_helper.make_script"?

Reply:
I considered this option, but I have doubt that this would make the code simpler. This code calls `dedent()` for string source, so you need a branching in any case. The rest of `make_script()` is not useful here. You would need to pass `os.curdir` as the first argument, the `.py` suffix is not relevant here, as well as the `importlib` cache invalidation.